src/master-playlist-controller.js

```diff
@@ -1068,11 +1068,10 @@ export class MasterPlaylistController extends videojs.EventTarget {
       return this.masterPlaylistLoader_.load(isFinalRendition);
     }
     // Blacklist this playlist
-    currentPlaylist.excludeUntil = Date.now() + BLACKLIST_DURATION;
+    currentPlaylist.excludeUntil = Date.now() + this.blacklistDuration;
```

Comment:
I think allowing blacklistDuration to be set in seconds by the user and converting to ms here would make this option easier to use